src/mainAppBar/mainAppBar.component.tsx

```diff
@@ -133,6 +134,12 @@ const MainAppBar = (props: CombinedMainAppBarProps): React.ReactElement => {
 
   const location = useLocation();
 
+  const navigateToAdminPage = (): void => {
+    props.navigateToAdminPage(
+      adminRoutes[props.adminPageDefaultTab ?? 'maintenance']
+    );
+  };
+
```

Comment:
This could be moved so it's defined entirely in the `mapDispatchToProps` using the second argument that gets passed to `mapDispatchToProps`: `ownProps` to access the default admin tab